src/inputs/Select/index.tsx

```diff
@@ -64,7 +80,16 @@ function Select({ items, activeItemId, onItemClick, id, ...rest }: Props) {
             return (
               <MenuItem value={i.id} key={i.id}>
                 {i.iconUrl && <IconImg alt={i.label} src={i.iconUrl} />}
```

Comment:
Is there a reason for using self-made image/text components instead of Material UI ones? `ListItemText` and `ListItemIcon`

https://material-ui.com/components/menus/#TypographyMenu.tsx

Reply:
I would avoid using Material for trivial components we can implement in two lines. Also, we have more control over it.